test/writer/model_reader_test.dart

```diff
@@ -66,6 +66,50 @@ TestModel:
       expect(simpleDateTime.type, isA<DateTimeType>());
       expect(simpleDateTime.isRequired, false);
     });
+
+    test('Test required not definable anymore', () {
```

Comment:
Also add tests for list & map with the correct config.

And also add test for the old config. -> that should expect an exception